app/src/main/java/com/mapbox/maps/testapp/examples/LocationComponentActivity.kt

```diff
@@ -100,6 +101,16 @@ class LocationComponentActivity : AppCompatActivity() {
         }
         return true
       }
+      R.id.heading -> {
+        binding.mapView.location2.updateSettings2 { puckBearingSource = PuckBearingSource.HEADING }
```

Comment:
nit: could we just use 
```suggestion
        binding.mapView.location2.puckBearingSource = PuckBearingSource.HEADING